.github/workflows/tests.yml

```
@@ -0,0 +1,155 @@
+---
+name: Tests
```

Also rename the file to `ci.yml`. WDYT?
```suggestion
name: CI
```

---

Sounds good to me.